backends/guest/include/common/util.h

```diff
@@ -52,6 +51,29 @@ struct signature_type_info {
 	size_t size;
 };
 
+/*
+ * Truncated form of a timestamp, fields are verbatim the same as in timestamp_t
+ */
+struct sv_trunc_timestamp {
+	leint16_t year;
+	uint8_t month;
+	uint8_t day;
+	uint8_t hour;
+	uint8_t minute;
+	uint8_t second;
+} __attribute__((packed));
+
+/*
+ * 8-byte header at the beginning of any variable,
+ *  containing a version field, and a truncated timestamp
+ */
+struct guest_variable_header {
+	uint8_t version;
+	struct sv_trunc_timestamp timestamp;
+} __attribute__((packed));
```

Comment:
This will give sizeof as 16?

Reply:
Also considering we don't use the version anywhere, don't see the reason to maintain a separate struct for it?